.github/workflows/ci.yaml

```diff
@@ -415,14 +431,18 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Give executable permissions to photofinish
         run: chmod +x $(whereis photofinish | cut -d" " -f2)
-      - name: Cypress run
+      - name: Parallel cypress
         uses: cypress-io/github-action@v6
         env:
+          SPLIT: ${{ strategy.job-total }}
+          SPLIT_INDEX: ${{ strategy.job-index }}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Where do those variables come out? Shouldn't be something like `strategy.matrix...` or `prepare.output.matrix...`?